components/camera/ffmpeg/ffmpeg.go

```diff
@@ -26,7 +26,6 @@ import (
 type Config struct {
 	CameraParameters     *transform.PinholeCameraIntrinsics `json:"intrinsic_parameters,omitempty"`
 	DistortionParameters *transform.BrownConrady            `json:"distortion_parameters,omitempty"`
-	Debug                bool                               `json:"debug,omitempty"`
```

Comment:
@randhid, I'm fairly certain this is safe? I wouldn't expect there to be an error if an existing config has `debug: true`. Do you know for sure if this is true? Happy to run an experiment, but I suspect you already have the answer.

Reply:
I have done this before in cowboy attribute deletion mode, yes a complete config removal is not breaking in my experience. Breaking config changes are the addition of a required field and the change of the name of an attribute that is required.

Right now people will have useless debug fields in their configs that do nothing. I'm okay with that.